packages/core/src/Chip/ChipContent/ChipContent.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space, fontSize } from 'styled-system'
+import { ArrowRight } from 'pcln-icons'
+import { ChipContentWrapper } from '../ChipContentWrapper'
+import { Flex } from '../../Flex'
+import { Text } from '../../Text'
+
+const ImageWrapper = styled(Flex)`
+  background-color: white;
```

Comment:
This should use `${getPaletteColor('background.lightest')}`

Reply:
Updated